### PR TITLE
Setup Specs

### PR DESCRIPTION
**Pull request by GrrrD**
This pull gets the tests setup for this repo and running on circleci. The main thing it adds is a regression test that converts the  Chirp_CCD.cda file and checks it against the ccd.hbs-Chirp_CCD.cda.json file. The latter file can be used to populate the CDP for running Inferno tests against. The Chirp_CCD.cda file should be kept up to date going forward whenever we modify the Chirp ccda output.

### CODE REVIEW

**Comment by GrrrD** [comment]
https://jira.onemedical.com/browse/PHM-541

File: src/lib/handlebars-converter/handlebars-helpers.spec.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ describe('Handlebars helpers', function () {
         });
     });
 
-    it('gzip returns an expected value', function() {
+    xit('gzip returns an expected value', function() {
```
Comment (misfo) [inline src/lib/handlebars-converter/handlebars-helpers.spec.js:246]: Should we add a comment for why this is skipped?

File: test/e2e-test/rule-test/util/test-rule-functions.spec.js [hunk 2]
```diff
@@ -231,18 +231,18 @@ describe('testRule', function () {
...
 
-    it('Rule officialValidator should return an object with valid status and empty string when the resource is valid.', () => {
+    xit('Rule officialValidator should return an object with valid status and empty string when the resource is valid.', () => {
         const reqJson = null;
         const resJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../test-samples/FHIR-R4/sample3-right.json')));
         assert.ok(testRules.officialValidator(reqJson, resJson).valid);
     }).timeout(300 * 1000);
 
-    it('Rule officialValidator should return an object with invalid status and error message when the resource is invalid.', () => {
+    xit('Rule officialValidator should return an object with invalid status and error message when the resource is invalid.', () => {
```
Comment (misfo) [inline test/e2e-test/rule-test/util/test-rule-functions.spec.js:245]: These too?

**Review by misfo — APPROVED** [review]
This is great that we can do end to end testing 🚀

File: src/lib/handlebars-converter/handlebars-helpers.spec.js (hunk 1 above)
Comment (GrrrD) [reply src/lib/handlebars-converter/handlebars-helpers.spec.js:246]: sure

**Review by misfo — DISMISSED** [review]

**Review by misfo — APPROVED** [review]

**Pull request merged**